WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -404,6 +406,40 @@ public void run() {
         ActivityId.trackLastActivity(ActivityId.POST_EDITOR);
     }
 
+    // this method aims at recovering the current state of media items if they're inconsistent within the PostModel.
+    private void resetUploadingMediaToFailedIfNotInProgressOrQueued(Context context, PostModel post) {
```

Comment:
oops! good catch - this was left there because it used not to belong to the Activity 😁 
addressed in 693b206